woocommerce/woocommerce-android
# Part 5: Signin M1 - add jetpack support documentation

This PR partially fixes #1091 by linking the new [Jetpack Setup Instructions](https://docs.woocommerce.com/document/jetpack-setup-instructions-for-the-woocommerce-mobile-app/) support document to the appropriate buttons in the new Sign in flow.

There are two ways to open this support document:
- Jetpack required screen: "View Instructions" button. 
- What is Jetpack dialog: "Learn more" button

<img src="https://user-images.githubusercontent.com/5810477/59400871-1d500c80-8d56-11e9-879a-b54e06713ccd.gif" width="300"/>


### To Test
- Login with a site that does not have jetpack installed. 
- Click "View Instructions" on the jetpack required screen - support page should open.
- Click "What is Jetpack?" on the jetpack required screen, then click "Learn more" - support page should open.
